docs/examples/good_practices/slurm_job_arrays/main.py

```diff
@@ -0,0 +1,180 @@
+"""Single-GPU training example."""
+import logging
+import os
+from pathlib import Path
+
+import rich.logging
+import torch
+from torch import Tensor, nn
+from torch.nn import functional as F
+from torch.utils.data import DataLoader, random_split
+from torchvision import transforms
+from torchvision.datasets import CIFAR10
+from torchvision.models import resnet18
+from tqdm import tqdm
+
+
+def main():
+    array_task_id = os.environ["SLURM_ARRAY_TASK_ID"]
+
+    training_epochs = int(array_task_id)
```

Comment:
C'est pas exactement ça qu'on veut changer d'une job à l'autre. Je pense qu'il y aurait de meilleures options:

- changer des paramêtres en les selectionnant dans une liste à l'index donné
- changer le random seed pour les mêmes paramêtres
- encore mieux: changer le random seed qui serait utilisé pour sample les paramêtres aléatoirement.

Je ne me souviens pas si l'exemple de base ajoute le random seed, mais sinon ce serait bien de le faire ici.

Reply:
Small comment here. If a user needs the `job.sh` to run the example it should still be found in the directory?